### PR TITLE
Molt-63 Filter Search Results

### PR DESCRIPTION
**Pull request by mag8224**
### Description
Added filtering options for the search results on the protein alignment 

### Checklist
- [x] Code compiles correctly
- [x] Created tests which fail without the change (if possible)
- [x] All tests passing
- [x] Extended the README / documentation, if necessary

### CODE REVIEW

**Review by Dennis-n — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
     "node": ">=10.1.0"
   },
   "scripts": {
-    "start": "webpack-dev-server --mode development --open",
+    "start": "webpack-dev-server --mode development --open --port 3000",
```
Comment (mev1482) [inline package.json:13]: Should this be changed? I thought right now this change is for localhost only.

Comment (arp6695) [reply package.json:13]: Good point, I don't think this should be deployed.

Comment (mag8224) [reply package.json]: Removed from PR

**Review by arp6695 — APPROVED** [review]
Verified.

**Pull request merged**